elliott/elliottlib/util.py

```diff
@@ -558,3 +558,17 @@ def log(msg):
         r = labels['release']
         all_payload_nvrs[component] = (v, r)
     return all_payload_nvrs
+
+
+def fix_summary_suffix(summary, summary_suffix):
+    # get summary of CVE tracker bug and update summary if needed
+    if '[' in summary and ']' in summary:
+        # Replace the first [..] with summary_prefix
+        start = summary.find('[')
+        end = summary.find(']', start)
+        new_s = summary[:start] + summary_suffix + summary[end + 1 :]
+    else:
+        # just append summary_suffix
+        new_s = summary + ' ' + summary_suffix
```

Comment:
f-strings can be used here

```suggestion
        new_s = f"{summary} {summary_suffix}"
```

Reply:
Done